internal/auth/testdata/krb5.conf

```diff
@@ -0,0 +1,20 @@
+[logging]
+ default = FILE:/var/log/krb5libs.log
```

Comment:
This may not work on all setups / Windows. I would suggest disabling logging for the test environment.